bin/nimbus-fml-configuration.sh

```diff
@@ -13,6 +13,9 @@ case "${CONFIGURATION}" in
     Fennec)
         CHANNEL="developer"
         ;;
+    Fennec_Enterprise)
```

Comment:
@jhugman is it fine if we add this configuration here? We need to use Fennec_Enterprise configuration and if we don't add it here the builds fail to compile with an error that the channel is not know (or something similar.. I forgot to take screenshot of the error)